src/miners/Algo3DCubeContiFrebet.cc

```diff
@@ -0,0 +1,32 @@
+#include <iostream>
+#include <Algo3DCubeContiFrebet.hh>
+
+bool
+Algo3DCubeContiFrebet::apply(GLfloat* vertices, GLfloat* colors, VertIndices& selected,
+                             size_t width, size_t height, size_t depth) {
+    make_vertices(vertices, width, height, depth);
+
+    const size_t chunk_size = 1024 * 100;
+    const u8* read = loader_->dataChunk(0, chunk_size);
+    std::cerr << "read " << chunk_size << " bytes!" << std::endl;
```

Comment:
We should probably use a proper logging tool with various log levels that can be turned on and off.
Not a blocker for now, and don't have enough time to allocate currently to block anything.